modules/builders-virtual-environment.adoc

```diff
@@ -91,8 +91,9 @@ oc sa get-token -n virtual-builders quay-builder
 +
 [source,terminal]
 ----
-$ oc create token quay-builder -n virtual-builders
+$ oc create token quay-builder -n virtual-builders --duration=<time_in_minutes> <1>
 ----
+<1> Optional. Specify the duration, to request a token with a custom expiration. The value and the unit are required. For example, specify `--duration 20160m` to retain the token for two weeks.
```

Comment:
@alishaIBM I discussed with @dmage to change as follows: 

```suggestion
$ oc create token quay-builder -n virtual-builders
----
+
[NOTE]
====
When the token expires you will need to request a new token. Optionally, you can also  add a custom expiration. For example, specify `--duration 20160m` to retain the token for two weeks.
====
```

Reply:
Thank you @SNiemann15. I have updated the PR.